docs/automating-using-elyra.md

```diff
@@ -0,0 +1,68 @@
+# Running a Kubeflow Workflow using Elyra
+
+## Background
+
+For our data science workflows, we use Jupyter notebooks for data collection, model training and all parts of the machine learning workflow. Some of those tasks are performed sequentially and continuously, requiring us to automate the sequential running of the notebooks using a simple workflow.
+
+Using the [Elyra](https://github.com/elyra-ai/elyra) JupyterLab and the notebook pipelines visual editor, we can create and run our notebooks using Kubeflow Pipelines, which is a platform for building and deploying scalable machine learning (ML) workflows.
+
+An instance of Kubeflow Pipelines is available for use in Operate First.
```

Comment:
```suggestion
An [instance of Kubeflow Pipelines](https://www.operate-first.cloud/users/apps/docs/kubeflow/kubeflow-pipelines/README.md) is available for use in Operate First.
```
Would it be worthwhile to add the link to the KFP readme on operate-first site here?